fern/pages/sdks/introduction/customer-showcase.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Customer Showcase
+---
+
+At Fern, we're proud to enable businesses to build and distribute world-class SDKs. From startups to industry leaders, our customers use Fern to streamline their development processes and enhance their developer experience. Below are just a few examples of companies that use Fern's capabilities to build and deploy their SDKs.  
+
+---
+
+<h2 style="margin-bottom: 0.25em">Square</h2>
+<span class="fern-docs-badge large green subtle rounded">public company</span>
+
+Square trusts Fern to generate and maintain **7 SDKs**, ensuring their APIs are easily accessible to developers building on any platform.  
+
+
+Check out Square's SDKs as they become generally available:
+<CardGroup cols={3}>
+  <Card title="Go SDK" icon="brands golang" href="https://github.com/square/square-go-sdk" />
+</CardGroup>
+
+---
+
+<h2 style="margin-bottom: 0.25em">Pinecone</h2>
+<div style={{ display: 'flex', alignItems: 'center' }}>
+<span class="fern-docs-badge large green subtle rounded" style="margin-right: 10px">gRPC</span>
+<span class="fern-docs-badge large green subtle rounded">REST</span>
+</div>
+
+Pinecone leverages Fern to create an SDK that supports **both gRPC and REST**, bridging the gap between broad compatibility and peak performance. 
+
+Check out Pinecone's SDK here:
+<CardGroup cols={3}>
+  <Card title=".NET SDK" icon="brands windows" href="https://github.com/pinecone-io/pinecone-dotnet-client" />
+</CardGroup>
+
+You read more about how Fern developed the first auto-generated SDK with gRPC and REST [in our blog](https://buildwithfern.com/blog/pinecone).
+
+---
+
+<h2 style="margin-bottom: 0.25em">ElevenLabs</h2>
+<div style={{ display: 'flex', alignItems: 'center' }}>
+<span class="fern-docs-badge large green subtle rounded" style="margin-right: 10px">file uploads</span>
+<span class="fern-docs-badge large green subtle rounded">websockets</span>
+</div>
+
+ElevenLabs uses Fern to generate SDKs with advanced capabilities for **file uploads** and **websocket communication**, enabling smooth interactions with their AI-powered audio APIs.  
+    
+Check out ElevenLabs' SDKs here:
+<CardGroup cols={3}>
```

Comment:
can we link to the right place within the SDKs to see either file uploads or websockets? could see splitting this into two different customers within the showcase or just specifying two links (file uploads & websockets)